modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm

```diff
@@ -322,13 +322,9 @@
 				hit_things |= slapped
 		if(!spinning)
 			break
-		addtimer(CALLBACK(src, .proc/animate_speen), 0.5 SECONDS)
-
-/mob/living/simple_animal/hostile/megafauna/gladiator/proc/animate_speen()
+		sleep(0.75)
 	animate(src, color = initial(color), 3)
-	addtimer(CALLBACK(src, .proc/stop_speen), 0.5 SECONDS)
-
-/mob/living/simple_animal/hostile/megafauna/gladiator/proc/stop_speen()
+	sleep(3)
```

Comment:
I really don't like the usage of sleeps, I guess we won't have a choice here, eugh.